README.md

```diff
@@ -132,7 +132,8 @@ You can configure `codeowners-generator` from several places:
 
 - **customRegenerationCommand** (`--custom-regeneration-command`): Specify a custom regeneration command to be printed in the generated CODEOWNERS file, it should be mapped to run codeowners-generator (e.g. "npm run codeowners").
 
-For more details you can invoke:
+- **check** (`--check`): It will fail if the CODEOWNERS generated doesn't match the current (or missing) CODEOWNERS . Useful for validating that the CODEOWNERS file is not out of date during CI.
+- For more details you can invoke:
```

Comment:
Don't wanna nitpick 😅 but this line should not be in the bullet list. It should be a standalone line, kind of the heading for the code snippet below.

Reply:
nitpick away. I'm not offended at all.

Reply:
you are also correct. why is this there at all